src/serac/physics/heat_transfer.hpp

```diff
@@ -771,16 +865,16 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
   mfem::Vector u_predicted_;
 
   /// Previous value of du_dt used to prime the pump for the nonlinear solver
-  mfem::Vector previous_;
+  mfem::Vector previous_u_dot_;
```

Comment:
👍

Reply:
I consolidated the naming a bit as suggested, but kept the different u and temperature names for now as they are used a bit differently.